portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path

```diff
@@ -53,6 +53,11 @@
     <td>xpath=(//input[contains(@class,'form-builder-page-header-title')])[${key_titleNumber}]</td>
     <td></td>
 </tr>
+<tr>
+    <td>PAGE_DESCRIPTION_FIELD</td>
```

Comment:
Hi @wesleyroberts , please use the already existent path `Form#DESCRIPTION_FIELD`. Please, remove this.